Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/Views/InputActionMapsTreeViewItem.cs

```diff
@@ -87,7 +101,7 @@ public void FocusOnRenameTextField()
 
             //a bit hacky - e.StopImmediatePropagation() for events does not work like expected on ListViewItems or TreeViewItems because
             //the listView/treeView reclaims the focus - this is a workaround with less overhead than rewriting the events
-            DelayCall();
+            schedule.Execute(() => renameTextfield.Q<TextField>().Focus()).StartingIn(120);
```

Comment:
Thank you for getting rid of that unreliable DelayCall

Reply:
Guess its still suffers from not being fully deterministic though...

Reply:
yes this delay could create some instabilities, it was hard to find somethings reliable 🤞